### PR TITLE
docs: add examples for multi-party workflows

### PR DESCRIPTION
**Pull request by m1ghtym0**
### Proposed changes
- Extends https://github.com/edgelesssys/marblerun/pull/342 with example workflows
- Do you find that helpful?
- Should we include the Coordinator logs ?

### CODE REVIEW

File: docs/docs/workflows/update-manifest.md [hunk 1]
```diff
@@ -52,17 +52,123 @@ Don't define other values except the `SecurityVersion` value for a package, as M
...
+    },
+```
+
+1. Alice applies an update by uploading an update manifest:
+
+    ```bash
+    $ marblerun manifest update apply new_manifest.json $MARBLERUN ---cert alice.crt --key alice-private.pem
+    Coordinator verified 
+    Loading client certificate
+    Creating client
```
Comment (daniel-weisse) [inline docs/docs/workflows/update-manifest.md:110]:
```suggestion
```
Debug statements that slipped through in a PR, will be removed by https://github.com/edgelesssys/marblerun/pull/356

File: docs/docs/workflows/update-manifest.md [hunk 2]
```diff
@@ -52,17 +52,123 @@ Don't define other values except the `SecurityVersion` value for a package, as M
...
+2. Bob checks the planned update:
+
+    ```bash
+    $ marblerun manifest update get $MARBLERUN
+    Successfully verified Coordinator
+    {
+        "Manifest": {
+            "Marbles": {
+                ...
+    }
```
Comment (daniel-weisse) [inline docs/docs/workflows/update-manifest.md:131]:
```suggestion
    $ marblerun manifest update get $MARBLERUN
    Successfully verified Coordinator
    {
        "Marbles": {
                ...
    }
```
https://github.com/edgelesssys/marblerun/pull/356 updates the command to return the exact bytes of the pending manifest

**Review by daniel-weisse — APPROVED** [review]

**Comment by thomasten** [comment]
Sorry for the delay. I like the examples! Pls review my last commit that fixes some nits in the examples and addresses the remaining issues of #342.

File: docs/docs/workflows/recover-coordinator.md [hunk 3]
```diff
@@ -63,18 +63,18 @@ Assume the following `RecoveryKeys` were set in the Manifest:
 
 1. The Coordinator returned the following `RecoverySecrets`:
```
Comment (daniel-weisse) [inline docs/docs/workflows/recover-coordinator.md:65]: Indentation is off in this listing. Two spaces for steps 1 and 2, and four spaces for step 3.
Not sure if it shows up on the webpage, but looks a bit weird in GitHub's markdown renderer

**Comment by m1ghtym0** [comment]
Thanks for the fixes, LGTM:-)

**Pull request merged**